Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -479,9 +479,10 @@
         },
         {
           "Id": "SEC101/050",
-          "Name": "DoNotExposePlaintextSecrets/IdentifiableNpmAuthorToken",
```

Comment:
>IdentifiableNpmAuthorToken [](http://example.com/codeflow?start=47&length=26)

The actual name of this rule is NpmAuthorToken, not sure how 'identifiable' was added. :)